BSB_LAN/BSB_LAN.ino

```diff
@@ -2906,13 +2909,18 @@ void LogTelegram(byte* msg) {
   int data_len;
   float dval;
   float line = 0;
+  if (LoggingMode & CF_LOGMODE_SD_CARD) {
 #if !defined(ESP32)
-  if (SDCard.vol()->freeClusterCount() < minimum_SD_size) return;
+    if (SDCard.vol()->freeClusterCount() < minimum_SD_size) return;
 #else
-  if (totalBytes()-usedBytes() < minimum_SD_size) return;
+    if (totalBytes()-usedBytes() < minimum_SD_size) return;
 #endif
+    dataFile = SDCard.open(journalFileName, FILE_APPEND);
+  }
 
+  bool is_query = false;
   if (bus->getBusType() != BUS_PPS) {
+    if (msg[4+(bus->getBusType()*4)]==TYPE_QUR) is_query = true;
```

Comment:
What about TYPE_QINF, TYPE_ACK, TYPE_NACK etc., do you want to log them as well? Do you know the state of `decodedTelegram` in these cases?